### PR TITLE
Add stars to profile

### LINKED ISSUE #4390: Display Star Ratings on Profile
As a funder on the Gitcoin platform, I'd like to see how contributors have been rated by other funders.

As a contributor on the Gitcoin platform, I'd like to see how funders have been rated by other contributors.

### What 

#4226
image

### Why
Increase in quality on the platform

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

add stars rating to profile
##### Refers/Fixes
#4390

### CODE REVIEW

File: app/dashboard/templates/profiles/profile.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
     <link rel="stylesheet" href={% static "v2/css/profile.css" %} />
     <link rel="stylesheet" href={% static "v2/css/tag.css" %} />
-    <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
+    <link rel="stylesheet" href={% static "v2/css/kudos/styles.css" %} />
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/profile.html]: lol

**Comment by owocki** [comment]
https://gitcoincore.slack.com/archives/CAXQ7PT60/p1558406373005800

**Comment by owocki** [comment]
i noticed the 'testing' field of this PR was left blank... next time it seems we need better testing

**Comment by octavioamu** [comment]
was tested I just didn't fill here, but with the 2 users I have in my local with data I didn't saw this problem.

**Pull request merged**